modules/cluster-logging-json-log-forwarding.adoc

```diff
@@ -1,7 +1,7 @@
 [id="cluster-logging-json-log-forwarding_{context}"]
 = Parsing JSON logs
 
-Logs including JSON logs are usually represented as a string inside the `message` field. That makes it hard for users to query specific fields inside a JSON document. OpenShift Logging's Log Forwarding API enables you to parse JSON logs into a structured object and forward them to either OpenShift Logging-managed Elasticsearch or any other third-party system supported by the Log Forwarding API.
+You can use a `ClusterLogForwarder` object to parse JSON logs into a structured object and forward them to either a supported output.
```

Comment:
I think either the word "either" should be dropped or you're missing another option here:
```suggestion
You can use a `ClusterLogForwarder` object to parse JSON logs into a structured object and forward them to a supported output.
```